.github/workflows/bandit.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   
     steps:
       - name: Clone the git repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
```

Comment:
Can we use 4.1.1 (https://github.com/actions/checkout/releases/tag/v4.1.1) already? I can see it's even used in a few places already...

Actually, (almost?) all dependencies have now newer major versions. I'm not sure, though, if you want (@wlemkows) to update these in this PR.

Reply:
I understand your point. Please for now let's keep these versions as they are because I don't want to struggle with the changes from the bot PR.

Reply:
👍